crates/levm/tests/tests.rs

```diff
@@ -1190,3 +1190,24 @@ fn mload_uninitialized_memory() {
     assert_eq!(loaded_value, U256::zero());
     assert_eq!(memory_size, U256::from(96));
 }
+
+#[test]
+fn pop_op() {
```

Comment:
Let's add a `pop_on_empty_stack` test case that should panic (after error handling this should be an error instead of a panic).

Reply:
Perfect, just added the test with a TODO comment of changing the panic